src/common/seed/images.ts

```diff
@@ -81,31 +81,18 @@ const generateRandomImage = () => {
 	return selectedVariant;
 };
 
-const remainingStatuses = ['building', 'error', 'ready']; //Is it correct way to make global variable in that function?
```

Comment:
I want to compare this file with master, not with your previous commit in a current branch. Please, squash your changes, you are definitely going to resolve merge conflicts, because you are touching exactly same files in different commits.